clouds/snowflake/modules/doc/h3/H3_POLYFILL.md

```diff
@@ -19,7 +25,7 @@ Returns an array with all the H3 cell indexes **with centers** contained in a gi
 
 ```sql
```

Comment:
Would maybe update:
**Example** -> **Examples** 

And add a couple more of examples. You can look at: core/clouds/bigquery/modules/doc/h3/H3_POLYFILL_MODE.md
For references.

Also one of the examples could exclude passing mode.